src/scripts/quests/QuestLineHelper.ts

```diff
@@ -2472,6 +2472,43 @@ class QuestLineHelper {
         App.game.quests.questLines().push(unrivaledPowerQuestLine);
     }
 
+    public static createLightsCameraPikaQuestLine() {
+        const lightsCameraPikaQuestLine = new QuestLine('Lights! Camera! Pika!', 'Frank is making a new movie, and needs your help with finding his stars.');
```

Comment:
This should probably have requirements for having Hawlucha and all the Pikachu needed for the quest. Also the lack of bulletin board makes this quest unpausable. I like the gift npc giving it out, but it's something we should note.

Reply:
It kinda fits the theme that you might need to catch the pokemon after you have started the quest, i think.
And did not want to put it on a bulletin board, as it's an anime questline, and should be a pixelhunt to be consistent.
But I can change it, if people think it should be changed

Reply:
Can always just add a new bulletin board in the pixel hunt town. There's no limit on having more of those, we just haven't had a reason to add more than 1 in a region until now

Reply:
It's now a bulletin board!